src/v3/features/congregation/settings/MeetingSection.tsx

```diff
@@ -0,0 +1,90 @@
+import useAppTranslation from '@hooks/useAppTranslation';
+import { CardSection, CardSectionContent, CardSectionHeader } from './CardSection';
+import { useState } from 'react';
+import { MenuItem, TextField } from '@components/index';
+import SwitchItem from './SwitchItem';
+import { LANGUAGE_LIST } from '@constants/index';
+
+const MeetingSection = () => {
+  const { t, i18n } = useAppTranslation();
+
+  const [autoCheckUpdates, setAutoCheckUpdates] = useState(true);
+  const [autoCheckFrequency, setAutoCheckFrequency] = useState('week');
+  const [language, setLanguage] = useState(i18n.language);
+
+  // Meeting schedules settings
+  const [displayExactDate, setDisplayExactDate] = useState(true);
+  const [useDisplayNameMeeting, setUseDisplayNameMeeting] = useState(false);
+  const [useDisplayNameOthers, setUseDisplayNameOthers] = useState(false);
+
+  // text formats
+  const [nameFormat, setNameFormat] = useState('firstLast');
+  const [dateFormat, setDateFormat] = useState('dateFormat1');
+
+  return (
+    <CardSection>
+      <CardSectionContent sx={{ gap: '16px' }}>
+        <CardSectionHeader title={t('tr_meetinMaterialsTitle')} description={t('tr_meetinMaterialsDesc')} />
+
+        <SwitchItem
+          value={autoCheckUpdates}
+          setValue={setAutoCheckUpdates}
+          label={t('tr_autoCheckUpdate')}
+          helper={t('tr_autoCheckUpdateDesc')}
+        />
+        <TextField
+          select
+          label={t('tr_autoCheckFrequency')}
+          value={autoCheckFrequency}
+          onChange={(e) => setAutoCheckFrequency(e.target.value)}
+        >
+          <MenuItem value="week">{t('tr_everyWeek')}</MenuItem>
+          <MenuItem value="twoWeeks">{t('tr_everyTwoWeeks')}</MenuItem>
+          <MenuItem value="month">{t('tr_everyMonth')}</MenuItem>
+        </TextField>
+        <TextField select label={t('tr_changeLanguage')} value={language} onChange={(e) => setLanguage(e.target.value)}>
```

Comment:
Sorry that this is not yet reflected in the design @Al11o, but we no longer need to specify the language for source materials and forms. It will always follow the app language with this update. Thank you.